hbase-server/src/test/resources/log4j2.properties

```diff
@@ -0,0 +1,137 @@
+#/**
```

Comment:
We have a log4j2.properties in the hbase-logging module, please modify that file if you want to do some trick in the log output for tests(but I do not think it is necessary?)

Reply:
This file has been added by mistake. Removed.